.gitmodules

```diff
@@ -28,4 +28,4 @@
 	url = https://github.com/cocodataset/cocoapi
 [submodule "third_party/cvcuda"]
 	path = third_party/cvcuda
-	url = https://github.com/banasraf/CV-CUDA.git
+	url = https://github.com/JanuszL/CV-CUDA.git
```

Comment:
Maybe instead of using private CV-CUDA forks we could put a specific branch in the CV-CUDA repo? I'm suggesting this in case anyone of us wouldn't be available to change something in this private fork

Reply:
I don't think we can put branch there. The path would be to use the currently used fork as a base and move to own fork then.